components/x-teaser-timeline/readme.md

```diff
@@ -0,0 +1,69 @@
+# x-teaser-timeline
+
+This component renders a list of articles in reverse chronological order, grouped by day, according to the user's timezone.
+It will optionally group today's articles into "latest" and "earlier" too.
+
+## Installation
+
+This module is compatible with Node 6+ and is distributed on npm.
+
+```bash
+npm install --save @financial-times/x-teaser-timeline
+```
+
+The [`x-engine`][engine] module is used to inject your chosen runtime into the component. Please read the `x-engine` documentation first if you are consuming `x-` components for the first time in your application.
+
+[engine]: https://github.com/Financial-Times/x-dash/tree/master/packages/x-engine
+
+## Other dependencies
+
+[o-teaser](https://registry.origami.ft.com/components/o-teaser) styles will need to be imported by the consumer of this component.
+
+If selectively importing o-teaser's styles via scss, then you will need the following:
+
+```scss
+$o-teaser-is-silent: true;
+@import 'o-teaser/main';
+@include oTeaser(('default', 'images', 'timestamp'), ('small'));
+``` 
```

Comment:
As per #221 I think this should link out to the x-teaser docs as they will always be up-to-date.